src/Dashboard/Dashboard.js

```diff
@@ -34,8 +35,8 @@ const Dashboard = (props) => {
   // Handle optional status property
   const noRun = scenarioState === 'Created' || scenarioState === null;
   const runInProgress = scenarioState === 'Running';
-  const hasError = scenarioState === 'Failed';
-  const isReady = scenarioState === undefined || scenarioState === 'Successful';
+  const hasError = scenarioState === 'Successful';
+  const isReady = scenarioState === undefined || scenarioState === 'Failed';
```

Comment:
Something is odd with these variables: the version before the change (`hasError = scenarioState === 'Failed';`) seems to be better than after the change (`hasError = scenarioState === 'Successful';`)

Reply:
It is a mistake. It is only to force and testing error diplay